runtime/onert/backend/ruy/KernelGenerator.cc

```diff
@@ -137,6 +137,8 @@ void KernelGenerator::visit(const ir::operation::FullyConnected &node)
   const auto bias_index{node.getInputs().at(FullyConnected::Input::BIAS)};
   const auto activation = node.param().activation;
   const auto weights_format = node.param().weights_format;
+  if (weights_format != ir::FullyConnectedWeightsFormat::Default)
+    throw std::runtime_error("Unsupported FullyConnected Weights Format");
```

Comment:
`weights_format` param is not used on `ruy` backend. `ruy` backend supports default format only.